scripts/setup_host.sh

```diff
@@ -14,6 +14,12 @@ CIV_WORK_DIR=$(pwd)
 CIV_GOP_DIR=$CIV_WORK_DIR/GOP_PKG
 CIV_VERTICAl_DIR=$CIV_WORK_DIR/vertical_patches/host
 
+FW_REL="linux-firmware-20220310"
+GUC_REL="70.0.3"
+HUC_REL="7.9.3"
+S_DMC_REL="2_01"
+P_DMC_REL="2_12"
```

Comment:
Please move these defs to ubu_update_fw() since they are not used out side of the func